moderntribe/wme
# Feature: wp admin colors refactor

Ticket: https://moderntribe.atlassian.net/browse/SITE-150

This adds the following features:

1. Adds a Service Provider implementation that works with the StellarWP container.
2. Tracks if a user has logged in for the first time (this tracks the state as of now, I can add additional logic/a cli command to set all existing users to have logged in, if needed).
3. If this is considered their first log in, set their admin color scheme.
4. Bugfix from https://github.com/moderntribe/wme/pull/201 to make e2e tests pass.

### QA

1. Pull down this branch.
2. Log in to the dashboard, and if your current color scheme isn't already set to the sitebuilder theme, it will be.
3. Future log in's after that initial one will no longer change the color scheme, if the user changed away from our theme after the fact.

![image](https://user-images.githubusercontent.com/1066195/214672372-8e161b70-d3c0-4c70-a577-1f4df10efdad.png)
